src/lib/holocene/table/paginated-table/index.svelte

```diff
@@ -1,34 +1,43 @@
 <script lang="ts">
   import type { HTMLAttributes } from 'svelte/elements';
 
+  import { twMerge as merge } from 'tailwind-merge';
+
   import SkeletonTable from '$lib/holocene/skeleton/table.svelte';
   import Table from '$lib/holocene/table/table.svelte';
 
   type Item = $$Generic;
 
-  interface $$Props extends HTMLAttributes<HTMLTableElement> {
+  type $$Props = HTMLAttributes<HTMLTableElement> & {
     visibleItems: Item[];
     loading?: boolean;
     updating?: boolean;
     maxHeight?: string;
     fixed?: boolean;
-  }
+    class?: string;
```

Comment:
I think the idea is to use the `ClassValue` type here, which supports arrays, objects, etc. but will defer to @GiantRobots

Reply:
`ClassNameValue` from the twmerge import is what we want (twmerge supports all of the cslx goodness... except objects. So we want to use what their api allows so we don't have to do more work subbing out to cslx, then merge, etc. objects are fine to not use because you can use arrays and you basically just invert the declarations)